src/test/java/net/minecraftforge/debug/misc/OnAnvilUpdateEventFix.java

```diff
@@ -0,0 +1,29 @@
+package net.minecraftforge.debug.misc;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import net.minecraftforge.event.AnvilUpdateEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod(OnAnvilUpdateEventFix.MOD_ID)
+@Mod.EventBusSubscriber(modid = OnAnvilUpdateEventFix.MOD_ID)
+public class OnAnvilUpdateEventFix 
```

Comment:
Please provide a javadoc for this class which explains what feature this tests, how to test it, and what the expected results are from said test.

Additionally, this test mod should be expanded to generally test the `AnvilUpdateEvent`' functionality, or removed until someone makes a general functionality test. If expanded, the name should then be `AnvilUpdateEventTest`.

Also, please cleanup the formatting in this class to remove trailing whitespace